projects/rocprofiler-systems/docker/Dockerfile.rhel

```
@@ -17,7 +17,7 @@ RUN yum groupinstall -y "Development Tools" && \
     yum install -y epel-release && crb enable && \
     yum install -y --allowerasing chrpath cmake curl dpkg-devel iproute libdrm-devel \
         ninja-build numactl-devel openmpi-devel papi-devel python3-pip sqlite-devel texinfo \
-        wget which zlib-devel && \
+        wget which zlib-devel json-devel && \
```

```suggestion
    yum install -y --allowerasing chrpath cmake curl dpkg-devel iproute json-devel \
        libdrm-devel ninja-build numactl-devel openmpi-devel papi-devel python3-pip \
        sqlite-devel texinfo wget which zlib-devel && \
```
Sorting list alphabetically.